dev/reference/tools/pd-control.md

```diff
@@ -45,13 +49,13 @@ Use TLS to encrypt:
 ### \-\-pd,-u
 
 + PD address
-+ Default address: `http://127.0.0.1:2379`
++ Default address: <http://127.0.0.1:2379>
```

Comment:
@CaitinChen I think you're working on this kind of URLs that may lead to 404. PTAL

Reply:
LGTM